sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableCatalog.java

```diff
@@ -71,6 +71,16 @@
    */
   String OPTION_PREFIX = "option.";
 
+  /**
+   * A reserved property to specify the version of the table.
+   */
+  String PROP_VERSION = "versionAsOf";
```

Comment:
Since we are using `version` and `timestamp` in the following `loadTable` methods, do you think it's better to simply use `version` and `timestamp` here for `PROP_VERSION` and `PROP_TIMESTAMP`?

Reply:
The reason I used `versionAsOf` and `timestampAsOf` is because Delta Lake uses these. 
Delta Lake doc has
```
df1 = spark.read.format('delta').option('timestampAsOf', '2019-01-01').load('/mnt/delta/people-10m')
df2 = spark.read.format('delta').option('versionAsOf', 2).load('/mnt/delta/people-10m')
```

Reply:
I see, that makes a lot of sense.